### PR TITLE
thread search component for desktop

### PR DESCRIPTION
**Pull request by mmaxim**
Storybook for the thread search component

image

### CODE REVIEW

**Comment by chrisnojima** [comment]
Is this going to be materially different on mobile. Would prefer these would be unified

**Comment by mmaxim** [comment]
Some key things are a little different, figured I'd start out this way and combine if possible when I got to mobile.

File: shared/chat/conversation/search/index.desktop.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    this._input && this.props.onSearch(this._input.getValue())
+  }
+  render() {
+    return (
+      <Kb.Box2 direction="horizontal" style={styles.outerContainer} fullWidth={true} gap="tiny">
+        <Kb.Box2 direction="horizontal" style={styles.inputContainer} fullWidth={true}>
+          <Kb.Box2 direction="horizontal" gap="xtiny" fullWidth={true} style={styles.searchContainer}>
+            <Kb.Icon type="iconfont-search" color={Styles.globalColors.black_50} fontSize={16} />
+            <Kb.Input
+              hideUnderline={true}
```
Comment (chrisnojima) [inline shared/chat/conversation/search/index.desktop.js]: Likely should use plain input

File: shared/chat/conversation/search/index.desktop.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    height: 14,
+  },
+  results: {
+    color: Styles.globalColors.black_40,
+  },
+  resultsContainer: {
+    flexShrink: 0,
+  },
+  searchContainer: {
+    alignItems: 'center',
```
Comment (chrisnojima) [inline shared/chat/conversation/search/index.desktop.js]: There are props for this

**Pull request merged**